packages/web/components/main-menu.tsx

```diff
@@ -0,0 +1,113 @@
+import Image from "next/image";
+import Link from "next/link";
+import Head from "next/head";
+import { useRouter } from "next/router";
+import { FunctionComponent } from "react";
+import classNames from "classnames";
+import { IS_FRONTIER } from "../config";
+import { useAmplitudeAnalytics } from "../hooks";
+import { MainLayoutMenu } from "./types";
+
+export const MainMenu: FunctionComponent<{
+  menus: MainLayoutMenu[];
+}> = ({ menus }) => {
+  const router = useRouter();
+  const { logEvent } = useAmplitudeAnalytics();
+
+  return (
+    <ul className="w-full flex flex-col gap-3 md:gap-0 mt-20 md:mt-0">
+      {menus.map(
+        (
+          { label, link, icon, iconSelected, selectionTest, amplitudeEvent },
+          index
+        ) => {
+          const selected = selectionTest
+            ? selectionTest.test(router.pathname)
+            : false;
+
+          return (
+            <li
+              key={index}
+              className={classNames(
+                "px-4 py-3 flex items-center cursor-pointer",
+                {
+                  "rounded-full bg-wosmongton-500": selected,
+                }
+              )}
+              onClick={
+                typeof link === "string" ? () => router.push(link) : link
+              }
+            >
+              <Head>{selected && <title key="title">{label}</title>}</Head>
+              <LinkOrDiv href={link}>
```

Comment:
opening a link in a new tab doesn'work. It seems to need to remove `Link` component if a menu has external link.